test/parallel/test-whatwg-url-searchparams-getall.js

```diff
@@ -25,15 +25,17 @@ test(function() {
 
 test(function() {
     var params = new URLSearchParams('a=1&a=2&a=3&a');
-    assert_true(params.has('a'), 'Search params object has name "a"');
+    assert_true(params.has('a'), `Search params object doesn't have name "a"`);
```

Comment:
We disable eslint for WPT tests though, because the styles are too different.. :D